### ISSUE TITLE
pfe-cta in grid layout

### ISSUE THREAD

**Issue by starryeyez024** [patternfly/patternfly-elements#976]
At the bottom of the demo page, it shows a grid layout. In this band, the active state on the secondary CTAs is undesirable so we may want to remove it  from the grid:
![image](https://user-images.githubusercontent.com/456863/87429976-8be34780-c5b2-11ea-9335-c3f749675907.png)

Also if it is a brick-like style, what's the suggested method for centering the text?
![image](https://user-images.githubusercontent.com/456863/87430024-9a316380-c5b2-11ea-84e0-f37b7be0fd7d.png)

**Comment by castastrophe** [comment]
I don't think this is about fixing the demo page, but rather, what is the desired style for a cta inside a layout that stretches it. We've tried to approach this before with max-content for example. I think we should explore that solution more.
